calico/etcddriver/protocol.py

```diff
@@ -150,6 +151,10 @@ def new_messages(self, timeout=1):
         try:
             data = self._sck.recv(16384)
         except socket.error as e:
+            #PLW: Should we return or retry in line? Concluded this is OK,
+            # but should comment that can return immediately with 0 messages
+            # in some error cases, which implies that you should retry.
+            # PLW: EAGAIN and EWOULDBLOCK are the same thing.
```

Comment:
I take it back - on Linux they are the same. On some other OSes they are not. Safer to treat them as not necessarily being the same, so leave code as is.